app/components/task-selection/component.js

```diff
@@ -128,25 +127,25 @@ export default class TaskSelectionComponent extends Component {
    * @property {Customer} _customer
    * @private
    */
-  @tracked
-  _customer = null;
+  @localCopy("args.initial.customer")
+  _customer = this.args.initial.customer || null;
 
   /**
    * The manually selected project
    *
    * @property {Project} _project
    * @private
    */
-  @tracked
-  _project = null;
+  @localCopy("args.initial.project")
+  _project = this.args.initial.project || null;
```

Comment:
Why`|| null` ?

Reply:
because it was like that before my changes

Reply:
it's not necessary any more I guess